docker/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN mkdir /build \
       && cmake -D CMAKE_BUILD_TYPE=release \
                -D TUVX_INSTALL_INCLUDE_DIR=/usr/local/include \
                -D TUVX_INSTALL_MOD_DIR=/usr/local/include \
+               -D LAPACK_LIBRARIES=-llapacke \
```

Comment:
@K20shores when I run the Fedora container on my local laptop with the same setup, everything builds fine. Do you have an idea why the `lapacke.h` can not be found in the CI workflow (https://github.com/NCAR/tuv-x/actions/runs/9719208263/job/26828579115?pr=97#step:5:308)?

Reply:
I might be out of the loop and missing something but that job points to an ubuntu github image based job (https://github.com/NCAR/tuv-x/actions/runs/9719208263/workflow?pr=97)?  So it looks like that job doesn't even use the Dockerfile but again, I may be missing where it's invoked.

Reply:
Ah, thanks @mwaxmonsky for pointing it out. I am looking at the wrong workflow.

But even for the Docker workflow, it still fails and it seems unable to find the `benchmark` folder (https://github.com/NCAR/tuv-x/actions/runs/9719208260/job/26828576416?pr=97#step:3:1306). Maybe somehow Adi's branch is not checked out correctly during the CI tests?

Reply:
It looks like we are now creating the build directory inside of the tuv-x source directory (ie. now we have `/tuv-x/build` instead of `/tuv-x` _and_ `/build`) and using a relative path to the source (`..` instead of `/tuv-x`).  Since the new path is inside the source directory, maybe changing it back would work?  I build like we are on this branch currently so it really shouldn't be a problem but I can't explain why it isn't working.

Reply:
Actually, I retract my previous statement.  It will probably still fail since the rest are unchanged and failing.  ☹️ 
Continuing to debug!